docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -1687,6 +1687,50 @@ <h1>Samza Configuration Reference</h1>
                     </td>
                 </tr>
 
+                <tr>
```

Comment:
This is the old configuration reference, please update the new samza-configurations.md instead.

Reply:
Done.